components/telemedicine/telemedicine-contact-card.tsx

```diff
@@ -0,0 +1,57 @@
+import { ChatAlt2Icon, ClockIcon } from "@heroicons/react/solid";
+import React from "react";
+
+import { PrimaryAnchorButton } from "~/components/ui/button";
+import { ContactDetail } from "~/lib/content/telemedicine-contacts";
+
+export function TelemedicineContactCard(contact: ContactDetail) {
+  return (
+    <div className="flex flex-col shadow-md rounded-md overflow-hidden">
+      <div className="flex justify-content-between">
+        <div className="flex-shrink-0 py-2 px-3 md:p-4">
+          <img
+            alt={`kontak dokter ${contact.doctor_name}`}
+            className="h-24 w-24 rounded-full"
+            src="https://i.pravatar.cc"
+          />
+        </div>
+        <div className="p-3 text-xs space-y-1 h-0 flex-1">
+          <p className="font-semibold text-gray-700">{contact.doctor_name}</p>
+          <div className="mt-2 flex justify-between w-full">
+            <p className="flex items-center text-sm text-gray-500">
+              <ClockIcon
+                aria-hidden="true"
+                className="flex-shrink-0 mr-2 h-4 w-4 text-gray-400"
+              />
+              {`${contact.ops_date} • ${contact.ops_time}`}
+            </p>
+          </div>
+          <div className="mt-2 flex justify-between w-full">
+            <p className="flex items-center text-sm text-gray-500">
+              <ChatAlt2Icon
+                aria-hidden="true"
+                className="flex-shrink-0 mr-2 h-4 w-4 text-gray-400"
+              />
+              {contact.platform}
+            </p>
+          </div>
+        </div>
+      </div>
+      <div className="p-3 justify-center">
+        <PrimaryAnchorButton
+          block
+          className="bg-light-blue-400 font-normal focus:bg-light-blue hover:bg-light-blue-600 text-white"
+          color="none"
+          data-testid={`contact-button-${contact.doctor_name}`}
+          href={contact.contact}
```

Comment:
the URL should be:


```js
const getLink = (platform, phone) => {
  if (platform.toLowerCase() === 'whatsapp') {
   // WA URL can not use 08, should use 628
    return 'https://wa.me/' + phone.replace(new RegExp(\^08\), 628)
  } else if (platform.toLowerCase() === 'telegram') {
    return 'https://t.me/' + phone
  } else {
    return '' // not available for Line
  }
}
```

Reply:
Thanks Mas for the code, haven't had time to do it yet 😅